Stats: Add Mobile Apps promo card via DotPager UI.

a8ck3n opened the pull request:
#### Proposed Changes

Adds the Jetpack mobile apps promo card alongside the Yoast SEO one using the `DotPager` component. The Yoast card is still the default.

Before:

![APromoSEO](https://user-images.githubusercontent.com/40267301/202101081-db395e83-8d1a-4208-8bc1-8654833efe99.png)

After:

![NewPromoCards](https://user-images.githubusercontent.com/40267301/203266516-9b427729-7a0b-437f-9bd6-3b22ecbed058.png)

#### Testing Instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

1. Visit the Calypso Live link.
2. Visit the Stats page.
3. Scroll down and confirm the `DotPager` card is present.
4. Confirm the Yoast SEO card is the default. 
5. Confirm the mobile apps card is available on the 2nd page.
6. Confirm it behaves appropriately on smaller screen sizes.

#### Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?
<!--
Link a related issue to this PR. If the PR does not immediately resolve the issue,
for example, it requires a separate deployment to production, avoid
using the "fixes" keyword and instead attach the [Status] Fix Inbound label to
the linked issue.
-->

Related to #69230 and supersedes #70059.

Currently just updating the Stats page. The designs call for the same promo card on the Insights and Ads pages but it looks overpowering there with the older page layout. I'm thinking we should wait until we "grid" those layouts before adding this component in.

a8ck3n committed:
Add Mobile Apps promo card via DotPager UI.

a8ck3n committed:
Add Mobile Apps promo card via DotPager UI.

msurdi-a8c committed:
Implement pricing plans block (#69918)

* Implement pricing plans block

* Add @wordpress/element to dependencies

* Open link to checkout in new tab

* Address linter warnings

* Use home_url instead of get_site_url

* Remove lodash

* Change BEM naming following code review recommendation

* Obtain option values in a more safe way following code review recommendation

* Add missing translations

* Follow the same css naming convention as the support-page happy block

* Remove extra newline

* Use existing variable for skeleton color

* Add constant for font-family

* Remove generic comment

* Fix typo

* Update saving computation

* Loop over plans data only once

* Re-implement pricing-plans hook to obtain plans info from @automattic/calypso-products

* Define configData only if not already defined

* Improve handling of inexistent/outdated product slugs

* Wrap block header/detail on small screens

* Use modern array syntax

* Fix typo

* Rename annually -> annual

* Rename billings -> planChoices

* Rename planBilling -> plan

* Open 'Learn more' link in new tab

* Baseline align pricing label and discount

* Align input label and promo text by line height

Co-authored-by: Matías Surdi <matias.emanuel.surdi@automattic.om>

arthur791004 committed:
Fix sass warnings (#70229)

manzoorwanijk committed:
Revert "Jetpack Cloud Pricing page | Update Jetpack Complete description (#70271)" (#70279)

This reverts commit c95a00c8f403bd0cd474083c85e77db22b9b15d1.

vasyas committed:
Add WP forum topic block (#70241)

* Initial version of Forum Topic embed

* wip

* Parse Forum Topic

* Show source

* Show details

* Store block source

* Update relative date on page load

* Corrected loading state for relative date

* Documentation fix

* Corrected block insert by selecting block type

* Pre-merge

Co-authored-by: vasyas <vasyl@stashuk.com>

keoshi reviewed (dismissed):
Hey, @a8ck3n — thanks for updating this. I noticed a few things happening:

- [x] In cases where the Yoast banner does not appear, we should not have the `DotPager` element and simply show the new banner.

![image](https://user-images.githubusercontent.com/390760/203300748-b3a5b1c0-8341-4e53-9a8c-f64933e9f247.png)

- [x] The wrapper card should have a white background.

![image](https://user-images.githubusercontent.com/390760/203300808-85f150c0-ecf3-4da1-b9a7-246bb88a9ee3.png)

- [x] On mobile, the wrapper card should behave like all others: lose the `border-radius` and `border-left` and `-right`.

<img width="418" alt="image" src="https://user-images.githubusercontent.com/390760/203301441-1e5e3557-b345-489c-8226-0ccd1bad5ffe.png">

- [x] On mobile, the QR should change to the corresponding app store button as proposed in p1HpG7-iJR-p2. We can probably use the buttons in the footer of this page as reference: https://apps.wordpress.com/mobile/

Before | After
--|--
<img width="344" alt="image" src="https://user-images.githubusercontent.com/390760/203302109-adb00e6e-0ccb-4ba4-973a-f4d39e16b8bc.png"> | <img width="738" alt="image" src="https://user-images.githubusercontent.com/390760/203302157-b5c209f3-f99c-4618-af49-89bbcf689fd5.png">

tyxla committed:
TS: Add missing package references (#70281)

jornp committed:
i18n: Fix RTL chevron in block editor sidebar home button (#68899)

dognose24 committed:
Stats: Leave the query date section for alignment of the period header (#70225)

heavyweight committed:
HC: fix promotional popover visibility when masterbar is hidden (#70282)

Commit:
Help Center: Change header icon style (#70031)

* HC: Change header icon style

* Fix icon on wp-admin

* Fix HC icon on different themes

* Fix icon on atomic and editor

dognose24 committed:
Stats: Refactor layout styles and adjust for the Store page (#70253)

* Refactor main chart styling layout on Traffic page

* Fix jetpack-backup-creds-banner styles

* Fix FormattedHeader styles on Ads page

* Extract layout styles as separate SCSS file

* Adjust Store page layout styles

* Refactor shared layout variables

sirbrillig committed:
Checkout: unify logic that determines if web payment methods are available (#70201)

* Update useStripePaymentRequest to check for both apple/google pay

* Update usePaymentRequestOptions/useStripePaymentRequest to allow null

* Update Apple/Google Pay to use updated useStripePaymentRequest

* Update checkout to use useStripePaymentRequest for allowed web types

* Remove unused useIsWebPayAvailable

* Change useStripePaymentRequest to use useStableCallback

* Change usePaymentRequestOptions to remove useTotal

* Allow generating paymentRequestOptions with a 0 total

mattwiebe committed:
Site Editor: handle dashboard navigation (#70301)

Gutenberg 14.5 has removed the slotfill we formerly used for the dashboard link override, instead introducing a `__experimentalDashboardLink` setting.

So we’re hacking in an event listener.

roundhill committed:
Target `body.is-videopress-stepper` to fix CSS targets in the intro view. (#70263)

jgcaruso committed:
remove tailored onboarding config check and config vars (#70300)

edanzer committed:
Data Stores: Add secondary WordPress domain to getSite selector (#69671)

roundhill committed:
Reverting #70263 because it broke the styles of other flows. (#70304)

taggon committed:
Add two new solutions to the cancellation flow (#69567)

* Add new solutions - Live Chat and Built By

* Fix wrong plan cost and refund.

* Delete obsolote tests

* Update Built By link

* Fix a bug that actionButtonUrl doesn't work

* Show open question form when solution is declined

* LiveChat is now enabled.

* Tweak styles for mobile devices.

* Open a livechat directly with a dummy text

* Show the cancellation reason in LiveChat

* Fix header's position

* Update the live chat feature

* Fix failed unit tests

roundhill committed:
Fixed dark intro bar styling on intro of VideoPress onboadring flow. Refs #70263 (#70308)

arthur791004 committed:
Pattern Assembler: Remove header and footer placeholder (#70276)

taipeicoder committed:
Theme Showcase: Revamp header (#70099)

p-jackson committed:
Add sites-as-landing-page feature flag (#70319)

a8ck3n commented:
Hey @keoshi, are you able to reproduce this?

> In cases where the Yoast banner does not appear, we should not have the `DotPager` element and simply show the new banner.

There isn't any logic in the code that manages when to show the Yoast banner. It should **always** be shown on the main Traffic page. My guess is that you encountered a JavaScript or network error during your testing that essentially broke the render.

If you can reproduce this, please let me know if there's anything listed in the console. In my testing, I see two console errors which shouldn't impact page rendering. Those are:

1. "Encountered two children with the same key, […]" — This is from the "?" mark help button in the top-right of the page.
2. "Invalid DOM property `stroke-width`." — This is from the SVG data and is being addressed [here](https://github.com/Automattic/wp-calypso/pull/70165).

Thanks!

a8ck3n committed:
Set the DotPager background to white.

a8ck3n committed:
Set the DotPager background to white.

a8ck3n committed:
Fix border styling at mobile sizes.

a8ck3n committed:
Fix border styling at mobile sizes.

a8ck3n commented:
Hello @keoshi! Items 2 & 3 from the above list have been addressed. As noted, I'm unable to reproduce item 1. If you can review your console output and let me know what you are seeing (and if the issue persists), that would be great.

Regarding the fourth item, showing the correct app store badge on mobile, I'll add that to this [separate PR](https://github.com/Automattic/wp-calypso/pull/70165) since that is handled in the component and not as part of the wrapper/container.

Thanks!

escapemanuele committed:
Domains and Plans steps in Stepper (#68722)

* Started to work on Domains step

* Domains Step loading

* Before fixing branch

* domain suggestion and save domain state

* Initial work

* Implement add to cart

* (wip) move plans to stepper

* domains and step merged

* New addPlan in cart

* processing works

* clean up and start to add style

* Signup Values in Onboard Store

* More clean up

* Add table plans style

* Add your domain, site transfer

* is site managed boolean

* Add better style on plans table

* Fix broken domains step

* fix initial query

* manage multi pending action conflicts

* No checkout if no cartitem

* Redirect after checkout

* Improve domains style

* Add import styles for domains

* Better domains style

* Add missing style for domains

* Set theme

* Add comingsoon

* Dont create site in domain step

* Fix small design issue on domains

* Add PlanSlug on onboard store

* Refactored processing step

* Removed multiple pending action leftovers

* no empty action in processing

* Fix hide free plan in plans

* Fix add plan to cart

* Check on checkout

* Added types

* Improve domains list style

* Props clean up for plans step

* Update productsList type

* Manage back from checkout

* link in bio type siteSlug

* Fix plans page refresh when change plan type

* Improve styling for plans toggle

* Fix merge mistake

* Improve domains style

* Add shopping Cart package

* Fix missing types in package

* Refactored type CartItem

* Decide Later domain fixed

* Fix load more button style for domains

* Processing step

* Improve plans style

* Fix mobile style on domains

* Add domain side content missing component

* Remove preCheckout && fix types

* Remove productsList deps

* Fix styling issue on side content domain

* Fix plans isDesktop function

* Fix style on side content for domains

* Click to use your domain

* Hide domain message on larger screens

* Revert temporary plans code change

* Dependency fix

* Missing yarn lock

* Checkout redirect if calypso live

* Try to fix calypso live

* Remove debugger

* Relative redirectTo

* Revert unnecessary domains changes

* Remove unnecessary code for plans

* Fix styling color

* Pass dynamic flow name to component and style fix

* Improve style and remove unnecessary code changes

* Removed window.redux

* Removed check for calypso live in checkout

* Cart client and removed calypso deps

* Remove unnecessary code for plans

* Remove wrong css classes for plans

* Missing createCartManager

* Remove duplicate

* Remove unused ts

* Use correct package import for plans

* Fix some types

* Add reference to ts project

* Fix types

* Optional booleans in DomainSuggestions

* Remove Marketing message

* LinkInBio: adjust navigation based on new URL

* Fix the signup header z-index & the getSite selector

* PlansStep: add missing loader styling

* Plans: fix undefined flowName

leading to lost feature list customizations for link-in-bio flow plans page

* Plans: fix custom header popular text for flows

* Plans: fix alignment of plan pill, header and price

* Plans Step: fix header background and margin

* Missing admin link in launchpad

* Fix gotoadmin missing on mobile

* Fix domains padding on mobile

* Fix eslint problem

* Fix plans styling

* Fix plans styling

Co-authored-by: Renan <renansscarvalho@gmail.com>
Co-authored-by: heavyweight <kpapazov@gmail.com>
Co-authored-by: Omar Alshaker <omar@omaralshaker.com>

jornp committed:
i18n: Add context for `Social` when referring to Jetpack Social (#68104)

a8ck3n commented:
This is addressed in PR 70165 [here](https://github.com/Automattic/wp-calypso/pull/70165).

> On mobile, the QR should change to the corresponding app store button as proposed in p1HpG7-iJR-p2.

keoshi commented:
> The first issue you noted, where the Yoast card is blank, is an odd one.

I think it might be just that I’m looking at Stats through a Jetpack site and the `PromoCardBlock` has some logic to check certain type sites and whether the banner is displayed or not:

https://github.com/Automattic/wp-calypso/blob/408c337c76209b2cb2c9334eccd1ef6e3762c9e4/client/blocks/promo-card-block/index.jsx#L42-L44

dognose24 committed:
Stats: Recover the chart with data for no resizing on the first load (#70288)

WBerredo committed:
Plugin Details: Add "Premium versions available" USP (#70312)

* Update the USP description color to gray-80

* Add an USP section for plugins with a premium version available

dognose24 commented on client/my-sites/stats/site.jsx:
```diff
@@ -406,20 +408,28 @@ class StatsSite extends Component {
-						image={ wordpressSeoIllustration }
-						href={ `/plugins/wordpress-seo-premium/${ slug }` }
-					/>
+				<div className="stats-traffic-promo-container">
```
```suggestion
				<div className="stats__promo-container">
```

According to the [naming guide of BEM](https://getbem.com/naming/), could we have this container with a more comprehensive name and append the modifier `--traffic` when necessary?

a8ck3n replied:
Thanks for the BEM link, Dognose. I reviewed that and the [Calypso CSS guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md) and I think I've got it right now. Let me know if you see anything else that isn't a best practice!

dognose24 commented on client/my-sites/stats/site.jsx:
```diff
@@ -406,20 +408,28 @@ class StatsSite extends Component {
-						href={ `/plugins/wordpress-seo-premium/${ slug }` }
-					/>
+				<div className="stats-traffic-promo-container">
+					<DotPager className="stats-traffic-promo-pager">
```
```suggestion
					<DotPager className="stats__promo-pager">
```

As what is mentioned above for BEM.

dognose24 commented on client/my-sites/stats/site.jsx:
```diff
@@ -406,20 +408,28 @@ class StatsSite extends Component {
-					/>
+				<div className="stats-traffic-promo-container">
+					<DotPager className="stats-traffic-promo-pager">
+						<div className="stats-content-promo-card">
```
```suggestion
						<div className="stats__promo-card">
```

As what is mentioned above for BEM.

jgcaruso committed:
VideoPress Onboarding: More style changes (#70305)

* use fancy apostrophe

* update signup form submit button style

* grey checkmark on domain selection `exact match` icon

* remove space in `not sure yet?`

* remove redundant `Upload videos` feature from list

* always show chevron for `upload your first video`

* add lines between launchpad items

Commit:
Fix GDPR banner styling issue (#70307)

karenroldan committed:
E2E: Update to wait for load event in account authentication flow (#70149)

* E2E: Update to wait for load event in account authentication flow

* E2E: Intercept route for login in authentication flow

lupus2k committed:
Gifting: Update gift label on checkout (#70309)

sharpirate committed:
Launchpad: Address a11y issues from aXe report (#70156)

* fix a11y issues on Launchpad

* set main section on launchpad

edanzer committed:
Newsletter Onboarding: Adjust color control on setup screen (#70160)

nelsonec87 committed:
Add Woo Tailored Onboarding (#70332)

* Added ecommerce flow with domains step

* Added default subtitle do domains step

* resetting onboard store after intro

blackjackkent committed:
Ecommerce Onboarding - Design tweaks (#70302)

* Design tweaks for ecommerce flow

* Fixing unintentionally global style change

arthur791004 committed:
Pattern Assembler: Adjust styles of the blank canvas card (#70321)

* Pattern Assembler: Set max-width on the description of the blank canvas card

* Adjust styles

* Remove space

miksansegundo committed:
Style selection in design picker: Background color of sidebar (#70274)

* Fix background on mobile

* Add variable

trakos committed:
Fix Jetpack Search pricing tier calculations (#70159)

Jetpack Search used pricing tiers data for some features.
However, with new pricing released recently, many of those
features stopped working.
New pricing uses quantity-based pricing, which requires
different type of calculations and more fields returned from API.
This PR required diff D92804-code, which added those fields.

List of fixes:
1. Fixed tooltip on https://cloud.jetpack.com/jetpack-search, it showed as "Up to 100 records 0 USD", now it will say "$8.95 per every additional 10k records and/or queries"
2. Added purchased quantity of requests/records allowed on a given Jetpack Search tier during checkout
3. Restored dropdown for switch between monthly and yearly billing for Jetpack Search

a8ck3n committed:
Update promo layouts for non-Atomic sites.

a8ck3n committed:
Update promo layouts for non-Atomic sites.

msurdi-a8c committed:
Implement minor tweaks, fixes and improvements for the new Pricing Plans block (#70330)

* Update enqueued script dependencies

* Update block category

* Fix typo

* Add radio options for pricing plan variants

* Limit amount of text shown in description

* Set correct 'Learn more' link

* Use selected support domain instead of current user's domain

* Fix loading animation colors

* Adjust billing period information to follow design specs

* Adjust settings controls to match designs

* Add workaround for 119-gh-Automattic/lighthouse-forums

* Load selected domain from existing post so it is available for edit mode in comments

* Include selected domain in block view configuration

* Create radio input groups by billing options instance

Co-authored-by: Matías Surdi <matias.emanuel.surdi@automattic.om>

arthur791004 committed:
Pattern Assembler: Update styles on the main screen (#70320)

* Update copy

* Adjust styles of the list

* Adjust button style

* Update title of the pattern selector

* Add icon back

a8ck3n committed:
Hide Yoast promo on non-Atomic Jetpack sites.

a8ck3n committed:
Hide Yoast promo on non-Atomic Jetpack sites.

simison committed:
Clean out stale isGutenboarding (#70327)

arthur791004 committed:
Stepper Framework: Quick fix on the navigation bar (#70362)

claudiucelfilip committed:
Fix: Logged-out navbar CTA parameter (#70328)

* Use current route for CTA param

* Use section name for CTA param

taipeicoder committed:
Theme Showcase: Show active theme as the first theme in the grid (#70326)

escapemanuele committed:
Exclude style from stepper (#70365)

a8ck3n committed:
Update CSS to better follow Calypso conventions.

a8ck3n committed:
Update CSS to better follow Calypso conventions.

a8ck3n commented:
- Added a check matching the one in the `PromoCardBlock` so that we can detect when we need the `DotPager` UI and when we don't.
- Cleaned up the class naming and CSS to better follow Calypso naming conventions.
- Tweaked the padding inside the `DotPager` to better match the original designs.

msurdi-a8c committed:
Add upgrade CTA clicks metrics (#70359)

Co-authored-by: Matías Surdi <matias.emanuel.surdi@automattic.om>

aneeshd16 committed:
BBE: Update copy on in-progress page (#70112)

* BBE: Update copy on in-progress page

* Add support link to content not submitted page

a8ck3n commented:
I'm renaming the `JetpackMobilePromoCard` to `MobilePromoCard` now that it handles both Jetpack and Woo. See related PR [here](https://github.com/Automattic/wp-calypso/pull/70165). Therefore, we need to hold this PR and update it to use the updated component name once #70165 is merged.

taggon committed:
Disable LiveChat for now (#70358)

fullofcaffeine committed:
[E2E][GB 14.6.x] Test fixes for Gutenberg (#70349)

* Scroll down to the bottom of the page before liking the post

Some posts might have content that makes the likes iframe to be below the fold. This seems to affect the iframe-getting code which can't find it. Scrolling to the bottom first fixes the issue.

* (e2e) `editorToolbarComponent#saveDraft()`: fix `saveDrafButton` selector for GB 14.6.0

The `aria-label` for the button now changes dynamically, depending on its state.

* Remove problematic `waitForNavigation` call

p-jackson committed:
Sites as landing page: redirect to `/sites` (#70356)

* Rename primary site slug variable

Now that we're dealing with multiple sites, I think it's clearer to be
explicit that `siteSlug` was referring to the primary site.

* Refactor landing page selection logic into its own function

Using a separate function allows early exit, which will make it easier
to only wait for data to load into the redux store when we have to.

* Use /sites as landing page based on user preference

* Add test harness given that banner isn't implemented yet

* Add tests for Calypso landing page

* Remove test harness

southp committed:
A quick and case-specific fix of handling the middle destination when (#70268)

the user step is removed.

southp committed:
Do not show the step progress for the dotlink variation of Link in Bio (#70270)

flow.

a8ck3n committed:
Stats: Fixes for the mobile promo card component (#70165)

* Fix strokeWidth value in SVG data.

* New story for Woo promo card.

* Add QR code PNG data and expose it to TS.

* Use the Woo QR code as appropriate.

* Use embedded PNG for Jetpack QR code.

* Remove block-level link around QR code.

* Rename class on QR code images.

* Update title & message text based on promo type.

* Display the Woo icon on the Woo card.

* Remove isMobile parameter.

* Add PNG data for Apple & Google stores.

* Add basic mobile device detection.

* Include iPads in device check.

It makes sense that we’d want the App Store badge on iPads too.

* Link up the store badges.

* Refactor title, message, and image logic for clarity.

* Refactor app store link setup.

* Cleaner PNG module declaration.

* Rename component to MobilePromoCard.

Now that is supports both Jetpack and Woo, this name makes more sense.

dognose24 commented:
The [blocker of this PR](https://github.com/Automattic/wp-calypso/pull/70165) has been merged! cc @a8ck3n @keoshi

taipeicoder committed:
Enable flag themes/showcase-i4/search-and-filter in Horizon (#70395)

jeyip committed:
Launchpad: Add notice to calypso sidebar (#70325)

taggon committed:
Add educational content to the cancellation flow (#69950)

* Fix wrong plan cost and refund.

* Fix a bug that actionButtonUrl doesn't work

* Open a livechat directly with a dummy text

* Show the cancellation reason in LiveChat

* Add educational content to the cancellation flow

* Add a link for a domain support doc

* Update images and content

* Reduce the gap above the CTA

* Visual update

* Reduce the header font size

* Fix a typo; fix visual style of a link

* Add a missing symbol.

* Improve LiveChat

* Fix CI errors

kangzj committed:
change /calypso/images -> calypso/assets/images to allow correct resolving (#70347)

aneeshd16 committed:
Launchpad: Use host from adminUrl for preview (#70367)

taipeicoder committed:
Theme Showcase: Add Tracks event (#70361)

niranjan-uma-shankar committed:
Replace decide_later with span so that we get a higher threshold for fuzzy match (#70396)

miksansegundo committed:
Pattern Assembler - Pattern improvements (#70364)

* Reeorder and add missing patterns

Revert

* Discard a pattern with a blog menu item

* Use sentence case

ciprianimike committed:
Update/cancellation modal feature list (#69682)

* add Cancellation Feature List

* remove get-subtitle-dialog

* update subtitle placeholder, remove debug code

* add starter plan cancellation feature list

* move feature to constants and styles for 'and more...' item

* getPlanCancellationFeatures naming convention

* refactor 'and more…' to be a flag

* cherry pick cancellation features

* put cancellation feature list constant in separated files

* fix typo

Commit:
`Favicons`: refactor tests to `@testing-library/react` (#69903)

* `Favicons`: refactor tests to `@testing-library/react`

* use `.toHaveAttribute` everywhere

ddc22 committed:
Prevent files being uploaded that are not of allowed types (#70369)

nelsonec87 committed:
Woo tailored flow: Added feature flag to redirect URL (#70394)

* Added flag to redirect URL

* Added params to URLSearchParams

eoigal committed:
Fix relative URLs in writing prompt notifications (#69949)

* Include host in writing prompt links

* use document.location.protocol

* Use Blogging Prompt terminology for consistency

* update css to use blogging prompts term

* Copy styles

* No need for styles outside of main

* need to handle host being widgets.wp.com on wpcom

* add debug to help figure out why links not working on wpcom

* Tidy up host lookup

* add comment

WBerredo committed:
Show the manage plugin option for plugins without a subscrition (#70393)

roo2 committed:
Add blogging prompt setting (#69848)

* add blogging prompt setting

* exclude mobile devices

* update comment to trigger tests

Co-authored-by: Eoin Gallagher <eoin@automattic.com>

SiobhyB committed:
Home: Update mobile card to display RTL logo when RTL locale is detected (#70439)

The update to the mobile card in #68062 includes the new "WP → JP" logo. By default, when viewed in RTL, this logo doesn't flow as expected for RTL locales. With this PR, the logo is updated and 'flipped' for RTL users when it's detected that an RTL locale is in use.

millerf committed:
Prevent edited plans to get influence from the flag #add-newsletter-payment-plan (#70368)

worldomonation committed:
Trim indent (#70387)

sirbrillig committed:
Payment Management: Add existing-card-ebanx payment processor to PaymentMethodSelector (#70265)

worldomonation committed:
test/e2e/specs/media/media__upload.ts (#70254)

- add comment directing investigators where to check in case of failure

vindl committed:
Themes: remove menu item from all sites views (#70440)

Themes don't support the multisite view currently and navigating to /themes
route will force you to pick a site. Considering that, we shouldn't show the
"View themes for all sites" nor the Themes link in the multisite sidebar.

taipeicoder committed:
Fix issue where body class is-section-themes-i4 is not added in the logged-out theme showcase (#70424)

taipeicoder committed:
Fixed issue where Jetpack sites would not properly load filter restuls (#70434)

bangank36 committed:
Update DP theme's copy (#70398)

*  Update copy of premium theme card
* Update copy for standard modal

niranjan-uma-shankar committed:
Remove retired experiment code (#70430)

niranjan-uma-shankar committed:
Remove login blue spec (#70422)

* Remove login blue spec

* Point to the regular login URL at wordpress.com/log-in instead of /log-in/new

a8ck3n committed:
Merge branch 'update/paged-promo-cards' of github.com:Automattic/wp-calypso into update/paged-promo-cards
